koku/api/report/queries.py

```diff
@@ -475,7 +490,8 @@ def __init__(self, provider, report_type):
         self._provider = provider
         self._report_type = report_type
 
-        self._map = ProviderMap.mapping
+        # FIXME: incorrect use of underscore convention.
+        # FIXME: these should be public.
```

Comment:
I'll fix this in a future PR. I'm leaving the comment in case someone else wants to do it before I get a chance.